lib/modules/platform/codecommit/index.spec.ts

```diff
@@ -49,7 +49,19 @@ describe('modules/platform/codecommit/index', () => {
 
   it('validates massageMarkdown functionality', () => {
     const newStr = codeCommit.massageMarkdown(
-      '<details><summary>foo</summary>bar</details>text<details>\n<!--renovate-debug:hiddenmessage123-->'
+      '<!--renovate:start--><details><summary>foo</summary>bar</details>text<details>\n<!--renovate-debug:hiddenmessage123--><!--renovate:end-->'
+    );
+    expect(newStr).toBe(
+      '**foo**bartext\n[//]: # (<!--renovate-debug:hiddenmessage123-->)'
+    );
+  });
+
+  it('validates massageMarkdown functionality with new lines', () => {
+    const newStr = codeCommit.massageMarkdown(
+      `<!--renovate:start-->
+<details><summary>foo</summary>bar</details>text<details>
+<!--renovate-debug:hiddenmessage123-->
+<!--renovate:end-->`
     );
     expect(newStr).toBe(
       '**foo**bartext\n[//]: # (<!--renovate-debug:hiddenmessage123-->)'
```

Comment:
it seems code commit supports hidden html comments. so we shouldn't remove them ? 🤔

Reply:
Judging from the logic: https://github.com/renovatebot/renovate/blob/ac4d310c6b2a239f1ed668f078ce8f6fa683422f/lib/modules/platform/codecommit/index.ts#L308